llvm/lib/Target/AArch64/AArch64LoadStoreOptimizer.cpp

```diff
@@ -965,6 +965,40 @@ static void updateDefinedRegisters(MachineInstr &MI, LiveRegUnits &Units,
       Units.addReg(MOP.getReg());
 }
 
+/// Find the DBG_INSTR_REF instruction that references the \p InstrNum
```

Comment:
```suggestion
/// Find the DBG_INSTR_REF instruction that references the \p InstrNum.
```

Reply:
Is there always at most one? If not, this should be "Scan for the first ... in MBB."